make pytest: You tried to register an orm model for typname elsewhere_with_participant, but no such type exists in the pg_type table of your database.

Hello. I've tried to run a Python test suite (`make pytest`) and there are 38 errors coming from `db.register_model(model)`, for example:

```
tests/py/test_api_key_json.py:7: in <module>
    from gratipay.testing import Harness
gratipay/testing/__init__.py:18: in <module>
    from gratipay.main import website
gratipay/main.py:64: in <module>
    website.db = gratipay.wireup.db(env)
gratipay/wireup.py:57: in db
    db.register_model(model)
env/lib/python2.7/site-packages/postgres/__init__.py:637: in register_model
    raise NoSuchType(typname)
E   NoSuchType: You tried to register an orm model for typname \
elsewhere_with_participant, but no such type exists in the pg_type \
table of your database.
```

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/26756959-make-pytest-you-tried-to-register-an-orm-model-for-typname-elsewhere_with_participant-but-no-such-type-exists-in-the-pg_type-table-of-your-database?utm_campaign=plugin&utm_content=tracker%2F85909&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F85909&utm_medium=issues&utm_source=github).
</bountysource-plugin>

--- a/Makefile
+++ b/Makefile
@@ -59,9 +59,9 @@ test-schema: env
 pyflakes: env
 	$(env_bin)/pyflakes *.py bin gratipay tasks tests
 
-test: test-schema pytest jstest
+test: pytest jstest
 
-pytest: env
+pytest: test-schema env
 	$(py_test) --cov gratipay $(pytest)
 	@$(MAKE) --no-print-directory pyflakes
 
@@ -78,7 +78,7 @@ node_modules: package.json
 	npm install --no-bin-links
 	@if [ -d node_modules ]; then touch node_modules; fi
 
-jstest: node_modules
+jstest: test-schema node_modules
 	node_modules/grunt-cli/bin/grunt test
 
 transifexrc:

